config/routes.rb

```diff
@@ -34,7 +34,9 @@
     end
     scope :module => "tournaments" do
       resources :invites, :only => [:show, :new, :create, :update]
-      resources :games, :only => [:index, :destroy]
+      resources :games, :only => [:index, :destroy] do
+        get :confirm
```

Comment:
Get requests shouldn't really have side effects. This would be better done as POST

Reply:
Doh will change to post stupid me